database/migrations/2016_06_07_153443_create_smart_lives_table.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class CreateSmartLivesTable extends Migration
```

Comment:
Oh. I didnt get this way ^^' sorry. Anyway maybe you're right, but smart life services are more like a product to sell rather than something similar to assistance services. For our app however, since it is so simple, maybe this division useless as you say. So if you want I can change my code as you suggested